template-parts/content-page.php

```diff
@@ -41,7 +41,7 @@
 							),
 						)
 					),
-					get_the_title()
+					esc_html( get_the_title() )
```

Comment:
wp_kses is already used here, on line 35.

Reply:
This instance of `get_the_title()` is outside `wp_kses()` used on line 35.  So I think it is still relevant to escape it.